Automattic/woocommerce-payments
# Design updates for Embedded Components

Fixes #9360

#### Changes proposed in this Pull Request





Note: we should hold off on merging this until the `dev/include-connect-js` PR is merged, and then target development.





This PR contains updates to the embedded onboarding flow to match the i4 designs: xO8YdpKfaVoP2d8NvweZ5I-fi-1095_43454

* Adds a border under the navbar.
* Updates the color and text of the button text on the nav bar to use theme colors.
* Update to the logo.
* Add left and right padding to add more separation
* Update the Stripe modal to use the Woo purple

#### Testing instructions





* Launch a new site with this branch using Jurassic Ninja ([quick link](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&woocommerce-payments-dev-tools&branches.woocommerce-payments=dev/i4-design-updates))
* Go to the WCPay Dev Tools on the new JN site and enable the `Embedded KYC` checkbox, then save the change.
* Start a new onboarding. You should see the Embedded KYC as part of the MOX flow.
* Verify the page looks as expected, and the designs match the figma.

Screenshots:

<img width="1506" alt="Screenshot 2024-09-04 at 16 22 46" src="https://github.com/user-attachments/assets/6a0a4ede-ecaa-4c98-a84f-e043435d1142">

<img width="1506" alt="Screenshot 2024-09-04 at 16 22 54" src="https://github.com/user-attachments/assets/6e5d9ce6-2f7b-4da5-a966-c1e8e7804569">

<img width="1499" alt="Screenshot 2024-09-04 at 16 24 01" src="https://github.com/user-attachments/assets/b5aa9183-7d53-4648-b0d1-54653eb300c0">


-------------------


**Post merge**
